marathon-bigip-ctlr.py

```diff
@@ -600,53 +750,84 @@ def create_config_marathon(bigip, apps):
         # The Marathon appId contains the full path, replace all '/' in
         # the name with '_'
         frontend_name = frontend_name.replace('/', '_')
-        f5_service['name'] = frontend_name
+
         if app.bindAddr:
             logger.debug("Frontend at %s:%d with backend %s", app.bindAddr,
                          app.servicePort, backend)
 
+        # pool members
+        members = []
+        key_func = attrgetter('host', 'port')
+        for backendServer in sorted(app.backends, key=key_func):
+            logger.debug("Found backend server at %s:%d for app %s",
+                         backendServer.host, backendServer.port, app.appId)
+
+            # Resolve backendServer hostname to IP address
+            ipv4 = resolve_ip(backendServer.host)
```

Comment:
This could be an IPv6 based on how I'm reading resolve_ip.  Is that OK?  Maybe name the var "ip"?